bin/dotnet

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+export ADO_TOKEN=$(azureauth --client 872cd9fa-d31f-45e0-9eab-6e460a02d1f1 --scope 499b84ac-1321-427f-aa17-267ca6975798/.default --tenant 72f988bf-86f1-41af-91ab-2d7cd011db47 --output token)
```

Comment:
should we consider having the prompt-hint option here?

Reply:
for telemetry reasons

Reply:
No telemetry is collected here ;)

Reply:
But if we get that sorted globally for ourselves, then it would be - so good call, I will add a prompt hint.